runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -44,6 +47,11 @@ J9::ClassEnv::getClassFromJavaLangClass(uintptrj_t objectPointer)
    }
 */
 
+J9::ClassEnv *
+J9::ClassEnv::self()
```

Comment:
I am not if the definition of self has to stay here, or in the OMR layer

Reply:
It's declared in `J9` namespace. It'd be probably better to stay in `OpenJ9` like `J9::CodeCache::self()`.